platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/state/signed/StartupStateUtilsTests.java

```diff
@@ -128,6 +135,30 @@ private SignedState writeState(
             final boolean corrupted)
             throws IOException {
 
+        final EventImpl[] events = new EventImpl[random.nextInt(1, 10)];
```

Comment:
why was this added?

Reply:
removed, good catch.  It must have been a stray copy-paste.